app/routes/models/organisation.js

```diff
@@ -34,7 +33,7 @@ export const getOrganisation = (request, organisation, errorText) => {
   const labelText = "Are these details correct?";
   return {
     backLink: {
-      href: requestAuthorizationCodeUrl(request),
+      href: 'TODO',
```

Comment:
reminder - this was probably supposed to be finished off

Reply:
Yep, sorted